packages/pn-commons/src/components/CustomTooltip.tsx

```diff
@@ -53,7 +53,7 @@ const CustomTooltip: React.FC<Props> = ({
           {...tooltipProps}
         >
           {cloneElement(children, {
```

Comment:
cloneElement is an expensive operation. we can avoid this if openOnClick is false

```
{openOnClick ? cloneElement(children, {
            onClick: handleTooltipOpen,
          }) : children}
```

Reply:
done